.github/commands/release-blocker.yml

```diff
@@ -0,0 +1,46 @@
+---
+trigger: release-blocker
+title: Release Blocker
+description: Describe the issue(s) preventing a release from being published
+surfaces:
+  - issue
+steps:
+  - type: form
+    style: embedded
+    body:
+      - type: textarea
+        attributes:
+          id: blocker-description
+          label: Description
+          placeholder: Describe the issue that prevents the release from being published.
+
+      - type: dropdown
+        attributes:
+          id: blocker-type
+          label: Type
+          options:
+            - 'Component or type removed'
+            - 'Type incompatability'
+            - 'Other'
+
+      - type: input
+        id: other
+        attributes:
+          label: Other
+          description: 'Provide a custom blocker type if other was used above'
+
+      - type: textarea
+        attributes:
+          id: pull-requests-to-revert
+          label: Pull Requests
+          placeholder: Provide one or more Pull Requests that need to be reverted in order to publish the release.
```

Comment:
I wonder if we can do release-blocker per PR?

For example just looking into the yesterday's issue (https://github.com/primer/react/issues/3455) Same PR has 2 blocking issues with different blocker types. 

If we can do per PR, we can maybe add `multiple: true` to the dropdown type attributes over [here](https://github.com/primer/react/pull/3463/files#diff-c17559b938b4bf025a7c87bff8120911af87d11349a27721609263b788c2f31cR18)?

Reply:
Ooo good call @broccolinisoup, will definitely add that in 👍